.circleci/config.yml

```diff
@@ -398,6 +398,9 @@ jobs:
       - run:
           name: generate repeat_writer_info
           command: cf run-task crt-portal-django  -c "python crt_portal/manage.py generate_repeat_writer_info" --name generate-repeat-writer-info
+      - run:
+          name: generate repeat_summary_info
+          command: cf run-task crt-portal-django  -c "python crt_portal/manage.py generate_repeat_summary_info" --name generate-repeat-summary-info
```

Comment:
yes, definitely

I started down that path (setting this file up so that the tasks were workflow based) but decided to hold off in this PR because it's kind of a large restructure of this file to make it happen cleanly.

I do intend to tackle this in a followup, though.

Long details: the problem is that while this is the current structure:

```mermaid
  classDiagram

  ProdExecutor-->ProdWorkflow
  ProdWorkflow-->ProdMaintenanceJobs
  class ProdMaintenanceJobs {
    LoginToProd
    Command1
    Command2
    Command3
  }

  StagingExecutor-->StagingWorkflow
  StagingWorkflow-->StagingMaintenanceJobs
  class StagingMaintenanceJobs {
    LoginToStaging
    Command1
    Command2
    Command3
  }
  
  DevExecutor-->DevWorkflow
  DevWorkflow-->DevMaintenanceJobs
  class DevMaintenanceJobs {
    LoginToDev
    Command1
    Command2
    Command3
  }
```

When we try to add a schedule for each command, because of the way circleci's yml works, it blows up a _lot_ in terms of code duplication:

```mermaid
  classDiagram

  ProdExecutor1-->ProdWorkflow1
  ProdWorkflow1-->ProdMaintenanceJobs1
  class ProdMaintenanceJobs1 {
    LoginToProd1
    Command1
  }

  ProdExecutor2-->ProdWorkflow2
  ProdWorkflow2-->ProdMaintenanceJobs2
  class ProdMaintenanceJobs2 {
    LoginToProd2
    Command2
  }

  ProdExecutor3-->ProdWorkflow3
  ProdWorkflow3-->ProdMaintenanceJobs3
  class ProdMaintenanceJobs3 {
    LoginToProd3
    Command3
  }
```

...for every command across every environment

Luckily, there's a way to define variables to reduce this duplication, but it's enough of a change that I'd want to try it out (and roll it out) separately from any other code change.

Reply:
That makes a lot of sense. Thanks!